CRM/Upgrade/Incremental/sql/4.7.17.mysql.tpl

```diff
@@ -7,3 +7,28 @@ UPDATE civicrm_navigation SET url = REPLACE(url, 'civicrm/tag', 'civicrm/tag/edi
 -- CRM-19815, CRM-19830 update references to check_number to reflect unique name
 UPDATE civicrm_uf_field SET field_name = 'contribution_check_number' WHERE field_name = 'check_number';
 UPDATE civicrm_mapping_field SET name = 'contribution_check_number' WHERE name = 'check_number';
+
+-- CRM-19715
+SELECT @closing_accounting_at := cov.value FROM civicrm_option_group cog
+  INNER JOIN civicrm_option_value cov 
+    ON cov.option_group_id = cog.id AND cog.name = 'activity_type' AND cov.name = 'Close Accounting Period';
+
+-- Delete all activities for Close Accounting Period
+DELETE FROM civicrm_activity WHERE activity_type_id = @closing_accounting_at;
+
+-- Delete Close Accounting Period activity type
+DELETE cov.* FROM civicrm_option_group cog
```

Comment:
I feel like we should to highlight to users as they upgrade that they will lose this data, but then I feel like the functionality hasn't been working to properly create these activities so it would be a distracting overkill message. Just noting in case someone else has an opinion during code review.